indico/web/templates/forms/form_common_fields_first.html

```diff
@@ -5,7 +5,7 @@
 {{ form_rows(form, skip=form._common_fields) }}
 {% call form_footer(form) %}
     <input class="i-button big highlight" type="submit" value="{% trans %}Save{% endtrans %}"
-           {% if disabled_until_change %}data-disabled-until-change{% endif %}>
+           {% if disabled_until_change|default(true) %}data-disabled-until-change{% endif %}>
```

Comment:
I don't think we ever pass `disabled_until_change` to this template... do we really want to start defaulting to true when the previous implicit default was falsy? We could just remove this altogether unless all the places where it's used are fine with disabled-until-change behavior

Reply:
I initially thought it would make more sense to have it `true`, but now that you mention it there might be indeed some occasions where we want to immediately save. I can see some usages of `disabled_until_change` being both `true` and `false`. So I can only think of defaulting it to false.